inc/class-sitemaps-posts.php

```diff
@@ -10,7 +10,7 @@ class Core_Sitemaps_Posts extends Core_Sitemaps_Provider {
 	 *
 	 * @var string
 	 */
-	protected $post_type = 'post';
+	protected $post_type = 'posts';
```

Comment:
The name of the post post_type is singular. (https://codex.wordpress.org/Function_Reference/register_post_type, see reserved post types),
but I suspect we are using this as a $name or $key really?

In any case this will break inc/class-sitemaps-posts.php:38

Reply:
Good point, I was using this for the name as well, I've updated to use `$name` everywhere I needed this instead